external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -74,6 +74,7 @@ class KafkaTestUtils(
   logInfo(s"Local host name is $localCanonicalHostName")
 
   private var kdc: MiniKdc = _
+  private var kdcDataDir: File = _
```

Comment:
Hm, quick question. Is this related to the fix? I couldn't follow why we need to have this var.

Reply:
consider `createKeytabsAndJaasConfigFile` as setup and `teardown` as cleanup - you'll find the reason. There might be some utility method to do cleanup of temp dir automatically as well.

Reply:
Thought the cleanup was not fully done and now I've fixed it. I've just double checked and `Utils.createTempDir()` adds this so the clean part is not needed: `ShutdownHookManager.registerShutdownDeleteDir(dir)`. Rolling back...